pkg/model/config/config.go

```diff
@@ -136,9 +205,13 @@ func (c Config) resourceAPIVersionCompatible(verType, version string) bool {
 		var currVersion string
 		switch verType {
 		case "crd":
-			currVersion = res.CRDVersion
+			if res.API != nil {
+				currVersion = res.API.Version
```

Comment:
```suggestion
				currVersion = res.API.Version 
```
/hold
we agreed on the meeting to keep the same name as the flag crdVersion in the project file. So, having the variable with a diff name is not helpful to improve the readability and its maintainbility.

Reply:
Same as in https://github.com/kubernetes-sigs/kubebuilder/pull/1869#discussion_r540080054